### PR TITLE
Refactor clean

### PR DESCRIPTION
**Pull request by mitchdraft**
* remove old code path (now using kubesquash method)

### CODE REVIEW

File: pkg/kscmd/debug.go [hunk 1]
```diff
@@ -110,53 +97,56 @@ func StartDebugContainer(config SquashConfig) (*v1.Pod, error) {
...
-				log.Warn(err)
-				dp.showLogs(err, createdPod)
-				return &v1.Pod{}, err
-			}
-		}
+	// Delaying to allow port forwarding to complete.
+	duration := time.Duration(5) * time.Second
+	time.Sleep(duration)
 
+	// TODO(mitchdraft) dlv only atm - check if dlv before doing this
```
Comment (yuval-k) [inline pkg/kscmd/debug.go]: I think we should do this TODO soon.
check dlv and offer alternative logic if. i suggest print the debug server url and let the user know that he should connect there.

File: PREVERSION.md [hunk 2]
```diff
@@ -28,57 +27,90 @@
...
-- tag a release update
-- push a new image to dockerhub
-- update the tag in `github.com/solo-io/squash/contrib/kubernetes/squash-client.yml`
+- [ ] (P0) update the docs
+  - [ ] (P0) no server required
+  - [x] new cli flags
+  - [ ] (P1) caveats for each debugger/ide
+- [ ] (P0) (small) tag a release update
+- [ ] (P0) (small) push a new image to dockerhub
+- [ ] (P0) (small) update the tag in `github.com/solo-io/squash/contrib/kubernetes/squash-client.yml`
```
Comment (yuval-k) [inline PREVERSION.md]: some more release tasks:
there are a few kubesquash commits that were contributed recently that we should port.

also port the CI process that releases everything automatically.
we probably want to merge the extensions (i.e. port the download an version pinning from the kube squash extension)

File: pkg/kscmd/debug.go [hunk 3]
```diff
@@ -80,25 +68,24 @@ func StartDebugContainer(config SquashConfig) (*v1.Pod, error) {
...
 	dp.getClientSet().CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: debuggerPodNamespace}})
 
 	createdPod, err := dp.getClientSet().CoreV1().Pods(debuggerPodNamespace).Create(dbgpod)
 	if err != nil {
-		return &v1.Pod{}, err
+		return nil, err
 	}
 
-	// TODO: we may be able to delete with DebugServer. see TODO below
-	if (!dp.config.DebugServer) && (!config.NoClean) {
```
Comment (yuval-k) [inline pkg/kscmd/debug.go]: why was this condition changed?

Comment (mitchdraft) [reply pkg/kscmd/debug.go]: good catch, trying to trim down on the redundant flags, but I forgot to apply the NoClean flag to the in-cluster config, fixed with https://github.com/solo-io/squash/pull/73/commits/37f3e2e11569103e081440903c20b4ab88deef31

File: PREVERSION.md (hunk 2 above)
Comment (mitchdraft) [reply PREVERSION.md]: added to the list

File: cmd/agent/main.go [hunk 4]
```diff
@@ -22,7 +20,7 @@ func main() {
 	customFormatter := new(log.TextFormatter)
 	log.SetFormatter(customFormatter)
 
-	log.Info(fmt.Sprintf("bridge started %v, %v", version.Version, version.TimeStamp))
+	log.Infof("bridge started %v, %v", version.Version, version.TimeStamp)
 
 	var err error
 	var cp platforms.ContainerProcess
```
[the remark this answers is not in the document]
Comment (mitchdraft) [reply cmd/agent/main.go]: good point, removed the contopid stuff but kept the fail on missing API, moved to a "must..." function

File: pkg/debuggers/controller.go [hunk 5]
```diff
@@ -189,104 +145,11 @@ func (d *DebugController) markAsAttached(namespace, name, debUrl string) {
...
 // uses the kubesquash debug approach
 func (d *DebugController) tryToAttachPod(da *v1.DebugAttachment) error {
 	ksConfig := kscmd.SquashConfig{
 		TimeoutSeconds: 300,
 		Machine:        true,
-		InCluster:      true,
-		DebugServer:    true,
 
 		DebugContainerVersion: version.ImageVersion,
 		DebugContainerRepo:    version.ImageRepo,
```
[the remark this answers is not in the document]
Comment (mitchdraft) [reply pkg/debuggers/controller.go]: planning to do this in a separate issue https://github.com/solo-io/squash/issues/74

File: pkg/kscmd/debug.go (hunk 1 above)
Comment (mitchdraft) [reply pkg/kscmd/debug.go]: https://github.com/solo-io/squash/issues/76

**Pull request merged**